packages/subgraph/src/mappings/TroveManager.ts

```diff
@@ -4,20 +4,8 @@ import {
   TroveLiquidated,
   Liquidation,
   Redemption,
-  BorrowerOperationsAddressChanged,
-  StabilityPoolAddressChanged,
-  CollSurplusPoolAddressChanged,
-  PriceFeedAddressChanged,
-  LQTYStakingAddressChanged,
-  LUSDTokenAddressChanged
```

Comment:
What’s the benefit of dropping this and adding the addresses to `subgraph.yaml`? Isn’t it more error prone? For instance if in some environment we have 2 deployments (not on mainnet so far, but it could happen locally or on a testnet), and for some reason one of the addresses is not changed, then the subgraph would be all messed up.

Reply:
I only used these events because we didn't have a template for the manifest, so I tried to hardcode as few addresses into the manifest as I could. That's no longer an issue, so it seems pointless to have all these boilerplate handlers in the code. It's impossible to get these events more than once anyway, because ownership is revoked upon setting them.

I'm not sure I get why having the addresses in the manifest is more error prone.

Reply:
> I'm not sure I get why having the addresses in the manifest is more error prone.

Precisely because of this:
> It's impossible to get these events more than once anyway, because ownership is revoked upon setting them.

:wink: 
With the events you make sure all addresses are consistent. The way it is now, you rely on the addresses on the manifest, and although we have a script to replace them, the source may be wrong (for instance the `dev.json` file). But anyway, no big deal, I’m aware those json files are automatically generated too, so I guess it’s fine as it is.

Reply:
I also realized that some constuctors emit events that the subgraph may be interested in (if not now, then in the future). A prominent example is the LQTYToken constructor, which mints the entire LQTY supply to a few addresses.

If we were to instantiate the data sources for these contracts from templates upon receiving some event from a TX on another contract, we might miss some crucial events.

Reply:
Yes, that’s a good point, actually that’s why I had to move out LQTY token out from the generic template to an entity in my original tokens PR.